rimage/manifest.c

```diff
@@ -1015,6 +1015,34 @@ const struct adsp machine_apl = {
 	.man_v1_8 = &apl_manifest,
 };
 
+const struct adsp machine_kbl = {
+	.name = "kbl",
+	.rom_base = ADSP_APL_DSP_ROM_BASE,
+	.rom_size = ADSP_APL_DSP_ROM_SIZE,
+	.sram_base = APL_DSP_BASE_ENTRY,
+	.sram_size = 0x100000,
```

Comment:
APL references in KBL/v1.5 code?

Reply:
Liam have confirmed that the address mapping of kabylake/skylake are the same as APL. OK, I will define a KBL one and keep it equal to APL.